ml-agents/mlagents/trainers/models.py

```diff
@@ -15,6 +14,19 @@
 EPSILON = 1e-7
 
 
+class CameraResolution(NamedTuple):
```

Comment:
nit: can we rename to something without "Camera" in it, since the observations could be generated a different way now?

Reply:
Renamed to `Tensor3DShape` - we might end up removing this tuple entirely if we do per-observation encoder types but for now it's still there.